rustls/src/server/server_conn.rs

```diff
@@ -147,6 +157,8 @@ impl<'a> ClientHello<'a> {
             server_name,
             signature_schemes,
             alpn,
+            server_cert_type,
+            client_cert_type,
```

Comment:
Should these new fields also get a `trace!` line above, alongside the `trace!` for sni, sig schemes, etc?